proctor-common/src/main/java/com/indeed/proctor/common/AbstractProctorLoader.java

```diff
@@ -52,6 +54,8 @@ public abstract class AbstractProctorLoader extends DataLoadingTimerTask impleme
 
     private final List<ProctorLoadReporter> reporters = new ArrayList<>();
 
+    private final Set<String> forcePayloadTests = new HashSet<>();
```

Comment:
I think it's better not to add `forcePayloadTests` here. It's used only by `proctor-pipet-core` which is a very different user of proctor library and cannot use forcePayload anyway.